src/components/home/Grade1to8.js

```diff
@@ -0,0 +1,272 @@
+import { makeStyles, IconButton, Divider } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+import React, { useContext } from 'react';
+import clsx from 'clsx';
+import { useTranslation } from 'react-i18next';
+import Drawer from '@material-ui/core/Drawer';
+import AppBar from '@material-ui/core/AppBar';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import MenuOpenIcon from '@material-ui/icons/MenuOpen';
+import { useTheme } from '@material-ui/core/styles';
+import Seo from '../common/Seo';
+import {
+  APP_AUTHOR,
+  APP_DESCRIPTION,
+  APP_NAME,
+  LEFT_MENU_WIDTH,
+} from '../../config/constants';
+import CollectionsGrid from '../collection/CollectionsGrid';
+import { QueryClientContext } from '../QueryClientContext';
+import { PUBLISHED_TAG_ID } from '../../config/env';
+import { PLACEHOLDER_COLLECTIONS } from '../../utils/collections';
+import Header from '../layout/Header';
+import Footer from '../layout/Footer';
+import SideMenu from '../layout/SideMenu';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+  },
+  appBar: {
+    zIndex: theme.zIndex.drawer + 1,
+  },
+  appBarBot: {
+    top: 'auto',
+    bottom: 0,
+    zIndex: theme.zIndex.drawer + 1,
+  },
+  drawer: {
+    width: LEFT_MENU_WIDTH,
+    flexShrink: 0,
+  },
+  drawerPaper: {
+    width: LEFT_MENU_WIDTH,
+  },
+  drawerHeader: {
+    display: 'flex',
+    alignItems: 'center',
+    padding: theme.spacing(0, 1),
+    // necessary for content to be below app bar
+    ...theme.mixins.toolbar,
+    justifyContent: 'flex-end',
+  },
+  content: {
+    flexGrow: 1,
+    padding: theme.spacing(3),
+    transition: theme.transitions.create('margin', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+    marginLeft: theme.spacing(5) - LEFT_MENU_WIDTH,
+  },
+  contentShift: {
+    transition: theme.transitions.create('margin', {
+      easing: theme.transitions.easing.easeOut,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+    marginLeft: theme.spacing(5),
+  },
+  mainWrapper: {
+    marginLeft: theme.spacing(5),
+    marginBottom: theme.spacing(6),
+  },
+  toolbar: theme.mixins.toolbar,
+  iconButton: {
+    padding: theme.spacing(1),
+  },
+  typographyMargin: {
+    margin: theme.spacing(1.5, 0),
+  },
+  divider: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+function Grade1to8() {
```

Comment:
The only difference between all these pages are the age category name? I think you could have created a whole component and provide the name as prop, and use it in the page directly.  I would have avoided lots of duplicated code.
```
<Component ageCategoryName="grade-1-to-8" />
```
This can also be a new issue.